runner/exiter.go

```diff
@@ -0,0 +1,13 @@
+package runner
+
+import "os"
+
+type Exiter interface {
+	Exit(code int)
+}
+
+type defaultExiter struct{}
+
+func (d defaultExiter) Exit(code int) {
+	os.Exit(code)
+}
```

Comment:
Maybe it could in internal to avoid exposing out of module

Reply:
done